js/whitebit.js

```diff
@@ -615,11 +708,443 @@ module.exports = class whitebit extends Exchange {
         return this.status;
     }
 
-    sign (path, api = 'publicV1', method = 'GET', params = {}, headers = undefined, body = undefined) {
+    async createOrder (symbol, type, side, amount, price = undefined, params = {}) {
+        let method = undefined;
+        await this.loadMarkets ();
+        const market = this.market (symbol);
+        const request = {
+            'market': market['id'],
+            'side': side,
+            'amount': this.numberToString (this.amountToPrecision (symbol, amount)),
+        };
+        const stopPrice = this.safeNumber2 (params, 'stopPrice', 'activationPrice');
+        if (stopPrice !== undefined) {
+            // it's a stop order
+            request['activation_price'] = this.numberToString (this.priceToPrecision (symbol, stopPrice));
```

Comment:
Here, line 723 as well as line 718, `priceToPrecision` and `amountToPrecision` both return strings, so we don't need numberToString.